esp-radio/src/wifi/mod.rs

```diff
@@ -3032,11 +3032,11 @@ impl WifiController<'_> {
     pub fn is_started(&self) -> Result<bool, WifiError> {
```

Comment:
Maybe we should adapt the docs stating this will return true if either STA or AP are started?

The current docs are not really wrong however - maybe just add a sentence what exactly is checked

Reply:
Good point